Makefile

```diff
@@ -121,6 +121,15 @@ rename-postgres-service: read-deployment-config ## make dev rename-postgres-serv
 remove-postgres-tf-state: terraform-init ## make dev remove-postgres-tf-state PASSCODE=XXX
 	cd terraform && terraform state rm cloudfoundry_service_instance.postgres
 
+restore-postgres: terraform-init read-deployment-config ## make dev restore-postgres DB_INSTANCE_GUID="<cf service db-name --guid>" BEFORE_TIME="yyyy-MM-dd hh:mm:ss" TF_VAR_apply_qts_docker_image=ghcr.io/dfe-digital/apply-for-qualified-teacher-status:<COMMIT_SHA> PASSCODE=<auth code from https://login.london.cloud.service.gov.uk/passcode>
```

Comment:
Shall we replace `make dev` with `make production`? In an outage situation, it may be easier to have the exact example to copy